src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -750,6 +766,45 @@ public function links()
     }
 
     /**
+     * Returns an Image object for the first node in the list.
+     *
+     * @param string $method The method for the image (get by default)
+     *
+     * @return Image An Image instance
+     *
+     * @throws \InvalidArgumentException If the current node list is empty
+     *
+     * @api
+     */
+    public function image($method = 'get')
```

Comment:
Exactly what I did in b0783b1fc29302a29f89194c5ed198be5f931f33, wanted to have your opinion on the commit before squashing.

I don't know about the `->link()` but it was already like this. I could add a commit to remove the method for `link` too, but I'm not sure it should be in this PR.